public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -926,6 +929,124 @@ $(function () {
+            } else {
+            $('#multi_select_modal #move_params').html("Move "+num_selected+" parameters");
+            }
+        } else{
+            $('#multi_select_modal #move_params').addClass("disabled");
+            $('#multi_select_modal #move_params').html("Move parameters");
+        }
+    })
+    $('#schema-builder').on('click', '.schema_group_move_params', function () {
+        // Get row
+        var row = $(this).closest('.schema_group');
+        var id = row.data('id');
+        launch_multi_select_modal(id);
+    });
+    function launch_multi_select_modal(id) {
+        // Reset everything to initial state
+        $('#multi_select_modal #move_params').addClass("disabled");
+        $('#multi_select_modal #move_params').html("Move parameters");
+        $('#multi_select_modal .params_table').show();
+        $('#multi_select_modal #no_params_alert').hide();
+        $("#search_parameters").val("");
+        // Build modal
+        $('#multi_select_modal .modal-header h4 span').html(id)
+        update_params_table();
+        $('#multi_select_modal').modal('show');   
+    }
+    // move selected parameters into the group, close modal if no top-level parameters are left
+    $('#move_params').click(function(){
+        var id = $('#multi_select_modal .modal-header h4 span').text();
+        var group_el = $('.schema_group[data-id="' + id + '"] .card-body');
+        var selected_params = $('#multi_select_modal').find('.select_param:checked');
+        for (let i = 0; i < selected_params.length; i++) {
+            var p_id = $(selected_params[i]).data('id');
+            var row_el = $('.schema_row[data-id="' + p_id + '"]');
+            group_el.append(row_el);
+        }
+        schema_order_change();
+        update_params_table();
+        $('#multi_select_modal #move_params').addClass("disabled");
+        $('#multi_select_modal #move_params').html("Move parameters");
+        if (!$('#multi_select_modal .params_table').is(":visible")){
+            $('#multi_select_modal').modal('toggle');   
+        }
+    });
+
+    // creates and updates the parameter table 
+    function update_params_table(){
+        $("#search_parameters").val("");
+        var params = '';
+        for (k in schema['properties']) {
+            // skip over Groups
+            if (schema['properties'][k].hasOwnProperty('properties')) {
+                continue
+            }
+            // create row for the table
+            params += `
+				<tr data-id=`+ k + `>
+                    <td><input type="checkbox" aria-label="Move this parameter" class="select_param" data-id=`+ k + ` id="group-move-` + k + `"></td>
+                    <td><label for="group-move-`+ k + `" class="text-monospace">` + k + `</label></td>
+                    <td><label for="group-move-`+ k + `" class="small">` + schema['properties'][k].description +`</label></td>
+                </tr>
```

Comment:
Missing the end of this escaped block, causing a JS error..

```suggestion
                </tr>`;
```